drivers/ukvirtio/bus/Makefile.uk

```diff
@@ -0,0 +1,10 @@
+$(eval $(call addplatlib_s,kvm,libukvirtio_bus,$(CONFIG_LIBUKVIRTIO_BUS)))
```

Comment:
This can be simplified a lot. The comments on other virtio devices apply here too, but this library must be conditional to `CONFIG_VIRTIO` instead, for reasons discussed on a previous comment:
```
+$(eval $(call addlib_s,libvirtio_bus,$(CONFIG_VIRTIO)))
+
+# common virtio headers
+LIBVIRTIO_BUS_ASINCLUDES-$(CONFIG_VIRTIO) += -I$(UK_DRIV_VIRTIO_BASE)/include
+LIBVIRTIO_BUS_CINCLUDES-$(CONFIG_VIRTIO)  += -I$(UK_DRIV_VIRTIO_BASE)/include
+
+LIBVIRTIO_BUS_SRCS-y += $(LIBVIRTIO_BUS_BASE)/virtio_bus.c
```